catalogapp/src/main/filtered-resources/WEB-INF/jsp/index.jsp

```diff
@@ -92,21 +118,21 @@ if(sec_roles != null) {
     <script type="text/javascript">
         document.getElementById('loading-msg').innerHTML = '<fmt:message key="loading"/>';
     </script>
-    
+
     <script type="text/javascript" src="lib/externals/ext/adapter/ext/ext-base.js"></script>
-    
+
     <!--
         loading custom parameters (see build profile)
     -->
-    <script type="text/javascript" src="app/js/GEOR_custom.js"></script>
+    <script type="text/javascript" src="<%= georCustomPath %>"></script>
```

Comment:
caution : this is not equivalent if georCustomPath  = "/app/js/GEOR_custom.js" (relative / absolute)